src/iotjs_binding.h

```diff
@@ -284,20 +311,25 @@ class JHandlerInfo {
   }
 
 #define JHANDLER_FUNCTION(name) \
-  static void ___ ## name ## _native(JHandlerInfo& handler); \
+  static void ___ ## name ## _native(iotjs_jhandler_t* jhandler); \
   static JRawValueType name(const JRawValueType func_obj_val, \
                             const JRawValueType this_val, \
                             const JRawValueType args_p [], \
                             const JRawLengthType args_cnt) { \
     JRawValueType ret_val = jerry_create_undefined(); \
-    JHandlerInfo handler(func_obj_val, this_val, &ret_val, args_p, args_cnt); \
-    ___ ## name ## _native(handler); \
+    iotjs_jhandler_t jhandler; \
+    iotjs_jhandler_initialize(&jhandler, func_obj_val, this_val, &ret_val, \
+                                         args_p, args_cnt); \
```

Comment:
Updated in [this commit](https://github.com/Samsung/iotjs/pull/485/commits/828c02bce5cd693fd3f622db01a6a1ce0cba61f1)